components/com_users/Controller/DisplayController.php

```diff
@@ -122,6 +122,14 @@ public function display($cachable = false, $urlparams = false)
 					$model = $this->getModel($vName);
 					break;
 
+				case 'users':
```

Comment:
Is this code really needed? Can you try it without this code block. Such code should be removed as part of this project instead of added as the core does model handling by its own, for most of the cases.

Reply:
I tried to remove it but I was able to make it work only when I set on the default case $model = $this->getModel($vName); instead of getting the Login model, I don't know if that is ok.

Reply:
Ok, then we leave it for now and we do some cleanup at the end, then we can try to get rid of this.